templates/zerver/help/user-groups.md

```diff
@@ -27,9 +27,22 @@ trying to send a message to a group of people, you'll want to either
 1. Add or remove users (including yourself). Click outside the box
    to save.  Zulip will notify everyone who is added or removed.
 
+{end_tabs}
+
+### Delete a user group
+
+{start_tabs}
+
+{settings_tab|user-groups-admin}
+
+1. Find the group.
+
+1. Click the trash icon (<i class="fa fa-trash-o"></i>) in the lower right corner of the group section.
+
+1. Click the Confirm button.
+
 !!! warn ""
-    **Note**: If you remove yourself from a user group, you
-    may no longer have permission to modify the user group.
```

Comment:
It looks like this warning was accidentally removed from the "Modify a user group" section and it should be added back to it.